vendor/arbiter/arbiter.cpp

```diff
@@ -2012,18 +2012,22 @@ S3::AuthFields S3::Auth::fields() const
     return S3::AuthFields(m_access, m_hidden, m_token);
 }
 
-std::unique_ptr<std::size_t> S3::tryGetSize(std::string rawPath) const
+std::unique_ptr<std::size_t> S3::tryGetSize(
+    const std::string rawPath,
```

Comment:
I assume that you'll want to alias tryGetSize with some default parameters for Headers and Query or you'll break the API, right?